media/css/firefox/home.scss

```diff
@@ -17,6 +17,23 @@ body {
     overflow-x: hidden;
 }
 
+.section-cta {
+    a:link,
+    a:visited {
+        color: #000;
+        display: block;
```

Comment:
Nit: making these links block level means they span their full container width. Whilst they occupy more click space this way, it could be a little confusing to click the empty whitespace by mistake:

<img width="575" alt="screenshot 2019-01-02 at 11 34 00" src="https://user-images.githubusercontent.com/400117/50590415-cbd85300-0e82-11e9-93d7-02ea2bc088d9.png">


Would setting them to `inline-block` instead be a slightly nicer experience?

Reply:
Agreed! 😄